doc/design.txt

```diff
@@ -272,7 +272,7 @@ purposes; they are not actually present in the boot vector.
     -----------------+--------+--------'
      swap: none                        |
     -----------------------------------'
-    
```

Comment:
There are 3 states that need special handling: `test`, `permanent` and `revert` and those are in the tables. There is one additional state which does nothing:  `none`. It just ends up with the image in slot 0 booting. This last one never gets to the swapping routines which employ the swap table you mentioned.